data_scripts/parseGenesToElastic.js

```diff
@@ -3,11 +3,11 @@ var level = require('level')
 var elasticsearch = require('elasticsearch')
 
 var client = new elasticsearch.Client({
-    host: 'localhost:9200',
+    host: 'elasticsearch:9200',
     log: 'info'
 })
 
-var db = level('/data/genenetwork/level/new/dbgenes_uint16be', {valueEncoding: 'binary'})
```

Comment:
this would break the current GN, better to perhaps make the location configurable, or to rename the metabrainnetwork folder to genenetwork when wanting to deploy with that version